events/network/file_activity.json

```diff
@@ -54,6 +54,11 @@
       "group": "primary",
       "requirement": "required"
     },
+    "session": {
+      "caption": "User Session",
```

Comment:
you can remove the `caption`, it is the same as the one in the dictionary, so it is redundant.

Reply:
Good thinking. I removed it.